tracing-subscriber/src/filter/layer_filters/mod.rs

```diff
@@ -470,6 +470,31 @@ impl<L, F, S> Filtered<L, F, S> {
     fn did_enable(&self, f: impl FnOnce()) {
         FILTERING.with(|filtering| filtering.did_enable(self.id(), f))
     }
+
+    /// Gives mutable access to the inner filter,
+    /// which can be used to change the filter
+    /// via the [`reload::Handle::modify`](crate::reload::Handle::modify) method.
+    ///
```

Comment:
how about:
```suggestion
    /// Mutably borrows the [`Filter`](crate::filter::Filter) used by this layer.
    /// 
    /// When this layer can be mutably borrowed, this may be used to mutate the filter.
    /// Generally, this will primarily be used with the 
    /// [`reload::Handle::modify`](crate::reload::Handle::modify) method.
    ///
    /// # Examples
```

Reply:
👍